### DISCUSSION TITLE
Unable to exit setup modus

### DISCUSSION THREAD

**Discussion by emilowich** [jomjol/AI-on-the-edge-device#2068] [Q&A] [answered]
I have finally sourced a esp32 cam with 8MB PSRAM, after many claimed 4MB which only had 2MB.
I have installed v14.0.3 and setup went fine, however every time I click the leave setup modus and reboot to normal mode, it reboots into setup modus. I have tried multiple times, I have checked the config.ini and it is setup mode = false, but it still reboots to setup mode every time. It is working and sending data to home assistant correctly, except for 1634,xxx being read as 1534,xxx. 
I am hoping I can use the set previous value to fix the digit reading, but alas, I cannot access it.
Hope someone has a solution, or just some suggestions to throw my way. Thanks

**Comment by kloodhu** [comment, marked as answer]
only a browser refresh (shift+f5) needed. now its okay.
